src/ImageSharp/PixelFormats/PixelBlenders/DefaultPixelBlenders.Generated.tt

```diff
@@ -78,32 +80,27 @@ namespace SixLabors.ImageSharp.PixelFormats.PixelBlenders
                 /// <inheritdoc />
                 public override TPixel Blend(TPixel background, TPixel source, float amount)
                 {
-                    return PorterDuffFunctions.<#=blender_composer#>(background, source, amount);
+                    TPixel dest = default;
+                    dest.PackFromVector4(PorterDuffFunctions.<#=blender_composer#>(background.ToVector4(),source.ToVector4(),amount.Clamp(0,1)));

```

Comment:
I think this is what I was referring to before. Should be `ToScaledVector4()` with the reverse `PackFromScaledVector4()` on repacking.

Reply:
Ahhh, okey, change it now